packages/urlwatch.rb

```diff
@@ -1,29 +1,29 @@
 require 'package'
 
 class Urlwatch < Package
-  description 'A tool for monitoring webpages for updates'
+  description 'urlwatch helps you watch changes in webpages.'
```

Comment:
No need to change the description here.  Which looks better after a search?
```
urlwatch: A tool for monitoring webpages for updates
```
or
```
urlwatch: urlwatch helps you watch changes in webpages.
```

Reply:
I like it when descriptions are full sentences. I'll figure out a combo between the two.